examples/example_parametrized_nodes.py

```diff
@@ -96,23 +96,18 @@ def objective(individual, seed):
 
 
 # %%
-# Next, we define the parameters for the population, the genome of
+# Next, we define the parameters for the genome of
 # individuals, the evolutionary algorithm, and the local search. Note
 # that we add the custom node defined above as a primitive.
 
-population_params = {"n_parents": 1, "seed": 818821}
+seed = 818821
```

Comment:
`seed = 1234`

Reply:
you mean bc it looks too hand picked? ;) should we change all seeds in the examples to 1234 then?

in general i think we can not avoid some handpicking just bc we want the examples to finish in reasonable time. a bad seed can prevent the search from finding a good solution. :(

Reply:
I just copied it from above, where you changed It in an example, however if possible we can just use the same everywhere?

Reply:
yes, it seems like we can use the same everywhere. just pushed a new commit where the seeds are all identical (except a few cases where we want to use multiple different seeds in the same script).